script.js

```diff
@@ -1,200 +1,82 @@
+                    kg: 0.001,
+                    lbs: 0.0022046244,
+                    oz: 0.03527
+                },
+                pound: {
+                    g: 453.592,
+                    kg: 0.453592,
+                    oz: 16
+                },
+                oz: {
+                    g: 28.3495,
+                    kg: 0.0283495,
+                    lbs: 0.0625
+                }
 };
 
-// third version of conversion function - used switch for tidier code
-const calculateTwo = () => {
-      let weight = input.value;
-        if (input.value === "") {
-              output.style.visibility = 'hidden';
-          } else
-            switch (weightType.value) {
-                    case `Kilograms`:
-                        output.style.visibility = 'visible';
-                        gramsOutput.innerHTML = weight*1000; 
-                        lbsOutput.innerHTML = weight*2.20462;
-                        ozOutput.innerHTML = weight*35.274;
-                        kgOutput.innerHTML = weight*1;
-                        break;
-                    case `Grams`:
-                        output.style.visibility = 'visible';
-                        gramsOutput.innerHTML = weight*1; 
-                        lbsOutput.innerHTML = weight*0.00220462;
-                        ozOutput.innerHTML = weight*0.035274;
-                        kgOutput.innerHTML = weight*0.001;
-                        break;
-                    case `Pounds`:
-                        output.style.visibility = 'visible';
-                        gramsOutput.innerHTML = weight*453.592; 
-                        lbsOutput.innerHTML = weight*1;
-                        ozOutput.innerHTML = weight*16;
-                        kgOutput.innerHTML = weight*0.453592;
-                        break;
-                    case `Ounces`:
-                        output.style.visibility = 'visible';
-                        gramsOutput.innerHTML = weight*28.3495; 
-                        lbsOutput.innerHTML = weight*0.0625;
-                        ozOutput.innerHTML =  weight*1;
-                        kgOutput.innerHTML = weight*0.0283495;
-                        break;
-                    default:
-                        return;
-                    }
-  };
-// 
-
-// EVENT LISTENERS \\
-// change placeholder text
-weightType.addEventListener('change', changePlaceholder);
-// conversions
-weightType.addEventListener('change', calculateTwo);
-// conversions for touch devices
-body.addEventListener('touchstart', calculateTwo);
-// show weight output
-input.addEventListener('input', calculateTwo);
-
-
+function toggleOutputDisplay() {
+  if (inputEl.value === "") {
+    outputEl.classList.add("hidden");
+  } else if (inputEl.value !== "") {
+    outputEl.classList.remove("hidden");
+  }
+}
 
+function changePlaceholder() {
+  inputEl.placeholder = `enter ${unitSelectorEl.value}`;
+}
 
+const conversionsOutput = () => {
+  toggleOutputDisplay();

```

Comment:
I necessarily think this is better than what you had but we can leave that for now